dio/lib/src/dio/dio_for_browser.dart

```diff
@@ -10,7 +10,7 @@ import '../response.dart';
 Dio createDio([BaseOptions? options]) => DioForBrowser(options);
 
 /// Implements features for [Dio] on Web platforms.
-class DioForBrowser with DioMixin implements Dio {
+class DioForBrowser extends DioMixin implements Dio {
```

Comment:
Should we make `DioMixin` as mixin instead?

Reply:
Yeah, I guess that makes sense. Not in this PR though.